analyses/snv-callers/README.md

```diff
@@ -5,7 +5,27 @@ This consensus mutation file is [MAF-like](#consensus-mutation-call) meaning it
 
 The resulting MAF and TMB files from this analysis are saved to `snv-callers/results/consensus/` folder and used by downstream analyses.
 
-See the comparison results plots [here](https://cansavvy.github.io/openpbta-notebook-concept/snv-callers/compare_snv_callers_plots.nb.html).
+Consensus mutations saved to `pbta-snv-consensus-mutation.maf.tsv.gz` are decided from agreement of [3 out of 3 of the snv callers used](#02-merge_callersr).
+Four callers were originally run but VarDict was dropped from downstream analysis because it called a large number (~39 million) of very low VAF mutations that were unique to it, suggesting that most of these were likely to be false positive calls.
+~187k mutations were called by all three of Mutect2, Strelka2, and Lancet.
+And between ~73k and ~1 million mutations were shared between combinations of any two these callers.
```

Comment:
Do these blurbs meet what you were thinking of, @jashapiro ?

Reply:
Yes, but these numbers seem wrong to me... We have 1.3M mutations in the consensus. Were you excluding the mutations from all 4 callers?

Reply:
Oh I see what you mean, the current comparison numbers do exclude that.